packages/react-reconciler/src/ReactFiberCommitWork.old.js

```diff
@@ -1725,6 +1728,7 @@ function commitWork(current: Fiber | null, finishedWork: Fiber): void {
             updateDeprecatedEventListeners(nextListeners, finishedWork, null);
           }
         }
+        prepareScopeUpdate(scopeInstance, finishedWork);
```

Comment:
Why is this necessary to have here? Shouldn't the call in complete work be sufficient?

As an aside, I *think* this would be the only place where we have an indirect call to `precacheFiberNode` in the commit phase.

Reply:
This is necessary so we update the the fiber on scope. We do one in the complete phase for initial mount, and in the commit phase after each update (so we point to the current fiber tree). This is similar to the `updateFiberProps` logic we have in the other commit methods, except scopes don't have props.